index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Portfolio: Setup and Mobile Version Skeleton</title>
+    </head>
+    <body>
+        <header class="toolbar">
+            <span>Welcome</span>
+            <button type="button" class="btn btn-tertiary"><img src="Images/Normal-Button/Tertiary/Icons/Union.svg" alt="union icon"></button>
```

Comment:
- [ ] Let's try to use the **semantic tags** as much as possible. Kindly enclose your hamburger menu with a `nav` tag.

Reply:
Hello, Thank you for your kind review, I have updated my project following your guidelines.